core/main/src/state/openrpc_state.rs

```diff
@@ -189,14 +188,36 @@ impl OpenRpcState {
         v.build_provider_relation_sets(&firebolt_open_rpc.methods);
 
         for path in extn_sdks {
-            if v.add_extension_open_rpc(&path).is_err() {
+            if v.load_ext_sdk_rpc(&path).is_err() {
                 error!("Error adding extn_sdk from {path}");
             }
         }
 
         v
     }
 
+    pub fn load_ext_sdk_rpc(&self, path: &str) -> Result<(), RippleError> {
+        info!("Loading extension OpenRPC from {path}");
+
+        let content = if path.contains("ripple-rpc.json") {
```

Comment:
I don't think `ripple-rpc.json` should have special handling, it should get loaded as any other extension schema. What's this for?

Reply:
@pahearn73 Reverted changes to use the new ripple-rpc.json provided.